### PR TITLE
Pin Task on Main Page.

### PR DESCRIPTION
**Pull request by MonCaptain**
Pinning a task on the main page now works. A task is displayed as pinned on both the sidebar and the main page. Additionally, if the user refreshes the page the task will still be there.

image

### CODE REVIEW

File: react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx [hunk 1]
```diff
@@ -29,28 +32,33 @@ export default function ({ todo, toggleComplete, removeTodo }) {
   function handleRemoveButton() {
     removeTodo(todo.id);
   }
+  
+  console.log("isActive",isActivePin)
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx]: console log?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
remove console log and solve merge conflict

File: react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx (hunk 1 above)
Comment (MonCaptain) [reply react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx]: can I do that within github, or do I go to the file in VS Code and recommit the changes?

Comment (Rebecca-CYLi) [reply react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx]: anyway you like, you just need to pull changes once merge into main branch

Comment (MonCaptain) [reply react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx]: yup, I got it

**Review by MonCaptain — COMMENTED** [review]
Nothing out of the ordinary

**Review by Rebecca-CYLi — APPROVED** [review]
LGTM

**Comment by MonCaptain** [comment]
> LGTM

Much thanks 😎

**Pull request merged**